lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -185,6 +185,10 @@ def kubectl
       @kubectl ||= Kubectl.new(namespace: @namespace, context: @context, logger: @logger, log_failure_by_default: false)
     end
 
+    def statsd_tags
+      %W(context:#{context} namespace:#{namespace} resource:#{id} type:#{type} sha:#{ENV['REVISION']} status:#{status})
```

Comment:
The "status" available here isn't the one you want; you need to extract the conditional too. 

Nit: can you move this to the `private` section?